charts/akeyless-secure-remote-access/templates/akeyless-zero-trust-portal/configmap.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
 data:
   env-config.js: |
     window._env_ = {
-        GW_DNS: {{ $.Values.apiGatewayURL | quote }}
+        CLI_GW_DNS: {{ $.Values.apiGatewayURL | quote }},
+        AUTH_DNS: 'https://auth.akeyless.io',
```

Comment:
You can put AUTH_DNS and the rest of the vars as default values with an option to change so other tenant could be applied

Reply:
@liorhasson should I leave it hardcoded as is or move it to values file with default ?

Reply:
Leave it, FE will add the option to work with other tenants